packages/react/src/lib/VoiceProvider.tsx

```diff
@@ -63,30 +49,34 @@ export type VoiceContextType = {
   isAudioMuted: boolean;
   isPlaying: boolean;
   messages: (
-    | UserTranscriptMessage
-    | AssistantTranscriptMessage
+    | Hume.empathicVoice.UserMessage
+    | Hume.empathicVoice.AssistantMessage
     | ConnectionMessage
```

Comment:
meeting note: This is manually written into our React SDK. In the future we may want to rely on auto-generated types, for now this will remain manual.